packages/terra-section-header/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-section-header",
+  "main": "lib/SectionHeader.js",
+  "version": "0.0.0",
+  "description": "Interactive header component that provides a status indicator.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-section-header",
+    "SectionHeader",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^2.1.0"
+  },
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-arrange": "^2.1.0",
+    "terra-base": "^3.1.0",
+    "terra-mixins": "^1.13.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "props-table": "props-table ./src/SectionHeader.jsx --out-dir ./docs/props-table",
```

Comment:
Remove `props-table` script. This change was updated in #1301

Reply:
Done, this should probably be logged as in issue to update it in [generator-terra-module](https://github.com/cerner/generator-terra-module)

Reply:
This was updated in the generator in https://github.com/cerner/generator-terra-module/pull/87

Reply:
Cool, I'll have to update my local generator then. I'd updated it before creating the package, but it seems it's been updated since then